setup.py

```diff
@@ -69,8 +69,8 @@
     "segmentation-models-pytorch==0.3.3",
     "ACSConv==0.1.1",
     # https://github.com/docker/docker-py/issues/3256
-    "requests<2.32.0",  # 2.32.0 are not compatible with docker 7.0.0; to remove restriction once docker is fixed
-    "docker",
+    "requests==2.32.0",  # 2.32.0 are not compatible with docker 7.0.0; to remove restriction once docker is fixed
```

Comment:
```suggestion
    "requests>=2.32.2",
```
I've checked different combinations and it seems that new docker is not working with `requests` `2.32.0 .. 2.32.1`, requiring `2.32.2` at least.